src/components/ansatte/RefreshToken.less

```diff
@@ -0,0 +1,7 @@
+.refresh-token-jwt {
```

Comment:
Ble vi enige om å følge standarden for css/less? Da burde den hete refreshtoken__jwt e.l?